bus-mapping/src/operation.rs

```diff
@@ -469,9 +469,9 @@ pub struct TxRefundOp {
     /// Transaction ID: Transaction index in the block starting at 1.
     pub tx_id: usize,
     /// Refund Value after the operation
-    pub value: Word,
+    pub value: u64,
```

Comment:
The value units here are `gas` right?  Could you please extend the comment to indicate that?